arch/arm/src/tlsr82/tlsr82_start.c

```diff
@@ -84,13 +84,13 @@ void __tc32_start(void)
   tlsr82_clock_init();
 
 #ifdef CONFIG_SCHED_BACKTRACE
-  extern uint32_t _sramcode;
-  extern uint32_t _eramcode;
+  extern uint8_t _sramcode[];
+  extern uint8_t _eramcode[];
   static void *g_code_regions[] =
   {
-    &_stext   , &_etext,
-    &_sramcode, &_eramcode,
-    NULL      , NULL,
+    _stext   , _etext,
+    _sramcode, _eramcode,
+    NULL     , NULL,
   };
 
   extern void up_backtrace_init_code_regions(void **regions);
```

Comment:
Maybe this ca be moved out of the function?

Reply:
Done.